applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/prj.conf

```diff
@@ -106,3 +106,30 @@ CONFIG_LOG_MODE_DEFERRED=y
 CONFIG_LOG_PRINTK=y
 CONFIG_LOG_BUFFER_SIZE=4096
 CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
+
+################################################################################
+# DFU configuration
+
+CONFIG_SUIT=y
+CONFIG_SUIT_DFU_CANDIDATE_PROCESSING_MINIMAL=y
+CONFIG_DESKTOP_DFU_MCUMGR_ENABLE=y
```

Comment:
Please move application modules and DFU transport configuration out of `DFU configuration` section. See `prj_fast_pair.conf` of `nrf52840dk` for reference.